domains/baa4ts.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "carlos-dev-g.is-a.dev",
+  "repo": "https://carlos-dev-g.github.io",
+  "owner": {
+    "username": "Carlos-dev-G",
+    "email": "baa4tsdev@gmail.com"
+  },
+  "record": {
+    "CNAME": "carlos-dev-g.github.io", "URL": "https://carlos-dev-g.github.io"
```

```suggestion
    "CNAME": "carlos-dev-g.github.io", "URL": "https://carlos-dev-g.github.io"
```
You cannot have both CNAME and URL records. Choose one or the other.